include/wlr/xwayland.h

```diff
@@ -179,6 +179,7 @@ struct wlr_xwayland_surface_configure_event {
 	struct wlr_xwayland_surface *surface;
 	int16_t x, y;
 	uint16_t width, height;
+	uint16_t mask;
```

Comment:
Can you add a little `xcb_config_window_t` comment to document where the bitfield values are coming from?

Reply:
Done.